x-pack/plugins/apm/server/lib/service_map/get_service_map_backend_node_info.ts

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { rangeQuery } from '../../../../observability/server';
+import {
+  EVENT_OUTCOME,
+  SPAN_DESTINATION_SERVICE_RESOURCE,
+  SPAN_DESTINATION_SERVICE_RESPONSE_TIME_COUNT,
+  SPAN_DESTINATION_SERVICE_RESPONSE_TIME_SUM,
+} from '../../../common/elasticsearch_fieldnames';
+import { EventOutcome } from '../../../common/event_outcome';
+import { environmentQuery } from '../../../common/utils/environment_query';
+import { withApmSpan } from '../../utils/with_apm_span';
+import { getProcessorEventForAggregatedTransactions } from '../helpers/aggregated_transactions';
+import { calculateThroughput } from '../helpers/calculate_throughput';
+import { Setup, SetupTimeRange } from '../helpers/setup_request';
+
+interface Options {
+  setup: Setup & SetupTimeRange;
+  environment?: string;
+  backendName: string;
+  searchAggregatedTransactions: boolean;
+}
+
+export function getServiceMapBackendNodeInfo({
+  environment,
+  backendName,
+  setup,
+  searchAggregatedTransactions,
+}: Options) {
+  return withApmSpan('get_service_map_backend_node_stats', async () => {
+    const { apmEventClient, start, end } = setup;
+
+    const response = await apmEventClient.search(
+      'get_service_map_backend_node_stats',
+      {
+        apm: {
+          events: [
+            getProcessorEventForAggregatedTransactions(
+              searchAggregatedTransactions
+            ),
```

Comment:
I think we always want ProcessorEvent.metric here

Reply:
Fixed in fa5fdbda0ed.